query_server/query/src/sql/planner.rs

```diff
@@ -804,7 +840,7 @@ impl<'a, S: ContextProviderExtension + Send + Sync + 'a> SqlPlanner<'a, S> {
     }
 
     fn show_databases_to_plan(&self, session: &SessionCtx) -> Result<PlanWithPrivileges> {
-        let projections = vec![col(DATABASES_DATABASE_NAME)];
+        let projections = vec![col(DATABASES_DATABASE_NAME).alias(DATABASES_DATABASE_NAME)];
```

Comment:
I'm a little confused about the modification here, the purpose of this?

Reply:
I want the column name to be in uppercase letters，same as before，but I forget add to_uppercase() function, sorr...